src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/ShardEndShardSyncStrategy.java

```diff
@@ -0,0 +1,53 @@
+package com.amazonaws.services.kinesis.clientlibrary.lib.worker;
+
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+public class ShardEndShardSyncStrategy implements ShardSyncStrategy {
```

Comment:
Does this class need to be public? Could you add javadoc to the changes.

Reply:
Changed it to package-private.